src/content/learn/react-developer-tools.md

```diff
@@ -1,33 +1,34 @@
 ---
-title: React Developer Tools
+title: Alat pengembang React
 ---
 
 <Intro>
 
-Use React Developer Tools to inspect React [components](/learn/your-first-component), edit [props](/learn/passing-props-to-a-component) and [state](/learn/state-a-components-memory), and identify performance problems.
+Gunakan Alat Pengembang React untuk memeriksa [komponen](/learn/your-first-component), menyunting [*props*](/learn/passing-props-to-a-component) dan [*state*](/learn/state-a-components-memory), dan mengidentifikasi masalah kinerja.
 
 </Intro>
 
 <YouWillLearn>
 
-* How to install React Developer Tools
+* Cara memasang Alat Pengembang React
 
 </YouWillLearn>
 
-## Browser extension {/*browser-extension*/}
+## Ekstensi Browser {/*browser-extension*/}
 
-The easiest way to debug websites built with React is to install the React Developer Tools browser extension. It is available for several popular browsers:
+Cara termudah untuk melakukan debugging website yang dibangun dengan React adalah dengan memasang ekstensi browser Alat Pengembang React. Ekstensi ini tersedia untuk beberapa browser populer:
 
-* [Install for **Chrome**](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en)
-* [Install for **Firefox**](https://addons.mozilla.org/en-US/firefox/addon/react-devtools/)
-* [Install for **Edge**](https://microsoftedge.microsoft.com/addons/detail/react-developer-tools/gpphkfbcpidddadnkolkpfckpihlkkil)
+* [Pasang untuk **Chrome**](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en)
+* [Pasang untuk **Firefox**](https://addons.mozilla.org/en-US/firefox/addon/react-devtools/)
+* [Pasang untuk **Edge**](https://microsoftedge.microsoft.com/addons/detail/react-developer-tools/gpphkfbcpidddadnkolkpfckpihlkkil)
 
-Now, if you visit a website **built with React,** you will see the _Components_ and _Profiler_ panels.
+Sekarang, jika Anda mengunjungi sebuah website **yang dibangun dengan React,** Anda akan melihat panel _Components_ dan _Profiler_.
 
 ![React Developer Tools extension](/images/docs/react-devtools-extension.png)
 
-### Safari and other browsers {/*safari-and-other-browsers*/}
-For other browsers (for example, Safari), install the [`react-devtools`](https://www.npmjs.com/package/react-devtools) npm package:
+### Safari and browser lainnya {/*safari-and-other-browsers*/}
+Untuk browser lainnya (misalnya, Safari), pasang paket npm [`react-devtools`](https://www.npmjs.com/package/react-devtools):
```

Comment:
```suggestion
### Safari and peramban lainnya {/*safari-and-other-browsers*/}
Untuk peramban lainnya (misalnya, Safari), pasang package npm [`react-devtools`](https://www.npmjs.com/package/react-devtools):
```

Reply:
`pasang package` or `instal paket`, according to the [Glossary](https://github.com/reactjs/id.react.dev/wiki/Glosarium#i)?

Reply:
instal _package_ according to Glossary

Reply:
`instal package` it is then

Reply:
I already push new commit that match with Glossarium. Take a look [here](https://github.com/reactjs/id.react.dev/pull/334/commits/b75c6f8898717cd2dfeae7a0eba510fa8e91f343)